src/parser.rs

```diff
@@ -77,25 +85,29 @@ fn get_tags(delim: &str, subject: &str) -> Vec<String>
 
 fn get_contexts(subject: &str) -> Vec<String>
 {
-    get_tags("@", subject)
+    get_tags(&REGEX_CONTEXTS, subject)
 }
 
 fn get_projects(subject: &str) -> Vec<String>
 {
-    get_tags("\\+", subject)
+    get_tags(&REGEX_PROJECTS, subject)
 }
 
 fn get_hashtags(subject: &str) -> Vec<String>
 {
-    get_tags("#", subject)
+    get_tags(&REGEX_HASHTAGS, subject)
 }
 
 fn get_keywords(subject: &str) -> (String, BTreeMap<String, String>)
 {
+    lazy_static! {
+        static ref REGEX: ::regex::Regex =
```

Comment:
Is it mandatory (by rust fmt or clippy, I don’t know) to use uppercase name for static local variable? For me, it’s for global variables.

Reply:
I get
```
warning: static variable `regex` should have an upper case name such as `REGEX`
```
if I don't put the name uppercase ;)

Reply:
(by the Rust compiler itself)